source/docs/structure.md

```diff
@@ -1,10 +1,15 @@
-title: Structure
+title: Application Structure
 ---
 
-Once initialised, here's what your project folder will look like:
+Once a [project is initialized](/docs/create_project.html), it's a good time to take a look at the resulting application structure, to get a better idea of what's going on. In this guide we'll discuss and explore an Embark's application structure and learn where to find different file types and configurations.
+
+Notice that this [guide covers](#Simple-template-structure) projects that have been created with the [`--simple` option](create_project.html#Creating-“contracts-only”-apps) as well.
+
+## Overview
+
+An Embark project's application structure is pretty straight forward. It contains a `config`, a `test`, an `app`, a `contracts` and a `dist` folder for their dedicated purposes. Most of them are self explanatory, but let's take a look at the overall file tree.
```

Comment:
Maybe here and above we can simplify a bit. Instead of sometimes saying "project", sometimes "application", sometimes both, should we settle on "project"?